zebra-consensus/src/chain/tests.rs

```diff
@@ -350,6 +250,12 @@ async fn verify_fail_add_block_checkpoint() -> Result<(), Report> {
     Ok(())
 }
 
+/*
+// This test is disabled because it doesn't test the right thing:
+// the BlockVerifier and CheckpointVerifier make different requests
+// and produce different transcripts.
```

Comment:
This test could be modified and re-enabled after we implement contextual verification in the state layer, so that we can make `CommitBlocks` requests.  However, we'd also need to change the chain it tests, because the block verifier cannot be used below sapling activation.

Reply:
Because we can't use any of the same blocks, and because we can't freely switch between checkpointed and block verification in the way that this test does, my feeling is that deleting this test and writing a different one later will be the least amount of work.